docs/section19/19-3.md

```diff
@@ -2,7 +2,7 @@
 **Schema Version:** 0.0.29
 **Mandatory Rule:** False    
 **Rule ID:** 19-3     
-**Rule Description:** Weather conditions used in sizing runs to determine baseline equipment capacities shall be based either on hourly historical weather files containing typical peak conditions or on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures.    
```

Comment:
@supriyagoel 
Supriya, @claperle  and I discussed this rule and saw that "hourly historical weather files containing typical peak conditions" is no longer an accepted option for weather in sizing runs in 90.1-2019. This was allowed in 90.1-2016 but seems was intentionally removed in 90.1-2019. We are proposing modifying this rule description and adding the FAIL case to the RDS as below.

Reply:
@JacksonJ-KC Yes, that makes sense to me. I have approved this RDS